client/lib/plugins/allowed-tags.js

```diff
@@ -0,0 +1,43 @@
+export const customTags = {
+	YOUTUBE: Symbol( 'youtube' ),
+};
+
+export const allowedTags = {
+	'#text': {},
+	a: {
+		href: true,
```

Comment:
Are we intentionally removing `target`? Won't this cause any external links to work like internal ones?

Reply:
It appears so. The intent was to keep the original [switch case](https://github.com/Automattic/wp-calypso/blob/trunk/client/lib/plugins/sanitize-section-content.js#L60) where only `href` was return true.